res/database/Default/database_news.xml

```diff
@@ -2,6 +2,2994 @@
 <tvtdb>
 	<version value="3" />
 	<allnews>
+
+		<!-- 'Real news reel' - Historical news sorted by date (some fictional 'alternative universe' variations may occur) -->
+
+			<!-- 1972-general -->
+
+		<news id="b5140fd5-e3af-40f4-a985-6d5b0a742271" type="0" creator="8936" created_by="jorgaeff" comment="https://en.wikipedia.org/wiki/Space_Shuttle_program | previously part of thread 5328557c-be27-46d2-acc0-a8ae2901eff3 | previously wrong date 1985">
+			<title>
+				<de>NASE startet Space-Shuttle-Programm</de>
+				<en>NASE launches space shuttle program</en>
+				<pl>NASE uruchamia program promów kosmicznych</pl>
+			</title>
+			<description>
+				<de>Gestern der Mond und morgen der Mars? Die nordamerikanische Raumfahrtbehörde NASE möchte auch im All die Nase vorn haben und startet ein neues Raumfahrtprogramm. Es soll der Erkundung neuer Planeten dienen.</de>
+				<en>The moon yesterday and Mars tomorrow? The North American space agency NASE wants to be in the lead also in space and launches a new space program. It is to help the exploration of new planets.</en>
+				<pl>Wczoraj Księżyc jutro Mars? Północnoamerykańska agencja kosmiczna NASE chce być liderem również w kosmosie i uruchamia nowy program kosmiczny. Ma on pomóc w eksploracji nowych planet.</pl>
+			</description>
+			<!-- flags: 2 = unique event, not auto-reuseable -->
+			<data genre="3" price="0.6" quality="60" fictional="0" flags="2" happen_time="4,1980,6,1" /> <!-- TODO: actually in 1972; 'adjust' once playing in the 1970s is possible -->
+		</news>
+
+			<!-- 1981-04 -->
+
+		<news id="9cbacfcc-7ad2-4623-b340-7286dd6fd5f5" type="0" thread_id="5328557c-be27-46d2-acc0-a8ae2901eff3" creator="8936" created_by="jorgaeff" comment="Columbia, Cape Canaveral - https://en.wikipedia.org/wiki/STS-1 | previously wrong date 1985">
+			<title>
+				<de>Kolumbus soll Erdumlaufbahn erkunden</de>
+				<en>Columbus to explore Earth orbit</en>
+				<pl>Columbus eksploruje orbitę okołoziemską</pl>
+			</title>
+			<description>
+				<de>Es ist der erste orbitale Flug eines unbemannten Space-Shuttles. Die Raumfähre Kolumbus soll den Start und die Landung eines Space-Shuttles erproben. Der Start erfolgt heute Nacht in Cap Carneval.</de>
+				<en>It is the first orbital flight of an unmanned space shuttle. The space shuttle Columbus is to test the launch and landing of a space shuttle. The launch will take place tonight in Cap Carneval.</en>
+				<pl>Jest to pierwszy lot orbitalny bezzałogowego promu kosmicznego. Wahadłowiec Columbus ma przetestować start i lądowanie promu kosmicznego. Start odbędzie się dziś wieczorem w Cap Carneval.</pl>
+			</description>
+			<data genre="3" price="0.65" quality="65" fictional="0" flags="2" happen_time="4,1981,4,12" />
```

Comment:
So that's more of a game design things. Playtesting, I don't think the fixed schedule "hurts" at all. It's not like players will start a new game in the same year every day and learn the news by heart. And even if, let them. The amount of "fixed" entries is still minuscule compared to the randomized news popping up, even in 1985. Sure, I can make the schedule more flexible (use parameter 5), but I don't feel it's really necessary.

> to be historically correct

Sure, lots of additional fictional things, but some real-historical things should roughly be correct, otherwise it's harder to maintain some consistency. For example, news item about a sports event, then the sports event in the programmes, then news item about the result, it should all flow together and work just by providing the correct dates in the data.

Also, with more historical news content, kinda being able to relive history could become another selling point for the game.